app/views/content/welcome/my-journey-into-teaching/_introduction.html.erb

```diff
@@ -1,36 +1,176 @@
+    <h2>What may surprise you?</h2>
+
+    <p>
+      I didn’t always love maths – it wasn’t until later in secondary school
+      that I discovered my potential.
+    </p>
+
+    <h2>It’s not always easy</h2>
+
+    <p>
+      The relationship that you have with the students is so special, and I did
+      struggle with remote learning during lockdown as it was harder to connect
+      with them. I missed them making jokes!
+    </p>
+
+    <div class="featured">
+      <h2>This year I'm looking forward to&hellip;</h2>
+
+      <figure>
+        <blockquote>Witnessing the progress of my students. I have such a great relationship with them and can't wait to spend more time in the classroom again.</blockquote>
+      </figure>
+    </div>
+  <% elsif subject_category == "science" %>
+    <h2>It all started with my own experiences</h2>
+
+    <p>
+      I really enjoyed the coaching aspect of the degree and getting to see the
+      kids improve. After uni, I did a few odd jobs from bathroom selling to
+      recruitment. But nothing had that ‘feel good factor.’ But then I took a
+      cover job in a local school and I fell in love with teaching.
+    </p>
+
+    <h2>The move from basketball to biology was easier than I thought</h2>
+
+    <p>
+      When the assistant head suggested teaching science, I wasn’t sure I was cut
+      out for it, I’d never been that academic myself. It turned out my degree
+      covered a lot of the science curriculum, so I decided to give it a go. I’ve
+      never looked back.
+    </p>
+
+    <h2>I discovered that science is all about making connections</h2>
+
+    <p>
+      I’m always showing my students how science relates to everything they’re
+      learning. From enzymes in cooking to the surprising history of a lot of
+      medical discoveries. I love it when they ask a question that I’ve never
+      even thought of.
+    </p>
+  <% elsif subject_category == "english" %>
+    <h2>It all started with my degree in sports science</h2>
+
+    <p>
+      Going to uni wasn't the norm where I'm from, but my teachers always
+      encouraged me. They made me believe that I could achieve anything. Plus,
+      I've always loved English. Believe me, I'm a proper library nerd.
+    </p>
+
+    <h2>I couldn’t wait to teach English</h2>
+
+    <p>
+      After uni, I completed my PGCE and quickly started teaching at an Academy.
+      I threw myself into the job and focused on developing a real reading
+      culture at the school. I knew that improving the literacy rates of every
+      student would make a huge difference, especially with kids who aren’t from
+      privileged backgrounds.
+    </p>
+
+    <h2>I realised that teaching English gives every student a voice</h2>
+
+    <p>
+      When a student can read and talk about their views, it opens them up to so
+      many opportunities. It’s all about having confidence, and I can actually
+      play a part in making that happen.
+    </p>
+
```

Comment:
Missing copy that needs adding in here

<h2>It’s not always easy</h2>
<p>I teach in a school where some kids come from quite difficult backgrounds. One girl I taught was convinced that she couldn’t study English, but now she’s doing incredibly well and has the GCSEs to prove it!</p>

<h2>I’ve risen through the ranks quickly</h2>
<p>I've gone from trainee teacher to head of English in less than 5 years. I had a bit of imposter syndrome at first, but I've quickly realised that even though it’s a big role, the focus on the students hasn’t changed at all.</p>